packages/components/organisms/f-checkout/src/components/_tests/Error.test.js

```diff
@@ -1,26 +1,56 @@
 import { VueI18n } from '@justeat/f-globalisation';
 import { shallowMount, createLocalVue } from '@vue/test-utils';
+import Vuex from 'vuex';
 import Error from '../Error.vue';
-import { i18n } from './helpers/setup';
+import { i18n, $cookies, $logger } from './helpers/setup';
+import { CHEKOUT_ERROR_FORM_TYPE, VUEX_CHECKOUT_MODULE } from '../../constants';
 
 const localVue = createLocalVue();
 
 localVue.use(VueI18n);
+localVue.use(Vuex);
+
+const mockState = {
+    basket: {
+        id: 'MGFkMzgwZjgtYjY3Yi00Nz-v1',
+        total: 0
+    }
+};
+
+const createStore = (state = mockState) => new Vuex.Store({
```

Comment:
Could you use the helper file for this? https://github.com/justeat/fozzie-components/blob/master/packages/components/organisms/f-checkout/src/components/_tests/helpers/setup.js#L104

Reply:
Fixed.